engine/src/physics/solver/integration/plugin.cpp

```diff
@@ -74,7 +77,7 @@ void cubos::engine::physicsIntegrationPlugin(Cubos& cubos)
                 velocity.vec += deltaLinearVelocity;
 
                 // Angular velocity
-                if (inertia.inverseInertia == glm::mat3(0.0F))
+                if (inertia.inverseInertia == glm::mat3(physicsConstants.minInvInertia))
```

Comment:
I'm not sure what this means, but is this really a constant that can be something other than zero?
Since you're using it only in `==` comparisons, it's not really a 'minimum inverse inertia', right? :thinking: 
But once again, I don't understand the math, so if it makes sense to be like this leave it as is

Reply:
Inertia has always been weird because the idea is that it is supposed to be checked in the same way as the normal mass which is just a float, but it's weird to do that with a matrix. Like do we set the minimum matrix and then do we want to check if the other has all elements smaller? Or just 1 element is enough? I'm not sure and that's why it's still this way. (reminder that it was like this because of the floating point "issues" which I'm not even sure if they're that relevant rn since we never really had any problems until now). If you have any idea for this, we'll take it.